docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture-use-cases.md

```diff
@@ -0,0 +1,163 @@
+
+### Multi-account
+
+In this architecture, there is a main VTEX account that will act as a [seller](https://help.vtex.com/en/tutorial/o-que-e-um-seller--5FkLvhZ3Few4CWWIuYOK2w) in all secondary VTEX accounts, acting as [marketplaces](https://help.vtex.com/en/tutorial/configurando-marketplace-entre-lojas-vtex). The main account - the seller - will be the source of truth for the [VTEX Core services](https://developers.vtex.com/docs/guides/getting-started#vtex-core-services), such as catalog, pricing, logistics, etc.
+
+The reference architecture below exemplifies its main characteristics:
+
+- **Separated VTEX account**: Each store has its own VTEX account.
+- **Separated website**: Each store has its own website and may customize its frontend with a different look and feel per store.
+- **Customer Data Storage**: Segregated by marketplaces' account. There is no customer information within the main account’s [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction).
+- **Catalog**: Separated catalog management, providing a unique assortment per store, defined based on the main account’s catalog.
+- **Logistics and Order Management System (OMS)**: Independent Logistics and OMS per store.
+- **Fulfillment**: Supports single or multiple fulfillment.
+- **Payments**: Independent Payments management per store.
+- **Pricing**: Independent Pricing management per store.
+- **Promotions**: Independent Promotions system per store.
+- **Message Center**: Independent Message Center per store
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/multi-currency-language-multi-account.png)
+
+#### Why choose this architecture?
+
+- The seller (main account) as catalog owner and marketplaces per country allows a way to make a multi-language solution, as each store can manage the language and currency locally, in its own store.
+- Each account maintains its own catalog information since the information coming from the main account is always in the standard language set there.
+- Support better translations compared to the [single account, multi-binding](#single-account-multi-binding) model, as there is an **independent catalog translation**. Therefore, you do not need to rely on Messages or Catalog translation app.
+
+>⚠ [Assembly options](https://developers.vtex.com/docs/guides/assembly-options-app), [attachments](https://developers.vtex.com/docs/guides/cart-attachments-section-api-quick-start-guides), and [services](https://developers.vtex.com/docs/guides/vtex-io-documentation-service) are not supported.
+
+Promotions from the main account have limited capabilities, as every information that comes from the customer will not be accessible by the main account. For example, if you want to create promotions for a customer cluster in the main account, it will not work because it does not have access to the marketplace’s Master Data.
```

Comment:
I would add this information elsewhere, not within "Why choose this architecture?"